pkg/operator/controller/names.go

```diff
@@ -76,6 +76,15 @@ func RouterOperatorGeneratedDefaultCertificateSecretName(ci *operatorv1.IngressC
 	}
 }
 
+// RouterPodDisruptionBudgetName returns the namespaced name for the router
+// deployment's pod disruption budget.
+func RouterPodDisruptionBudgetName(ic *operatorv1.IngressController) types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: "openshift-ingress",
+		Name:      "router-" + ic.Name,
```

Comment:
using the word "router" in any of our resources was a mistake from the start — could we avoid propagating it further?

Reply:
We could, but this would be the odd resource out till we renamed other resources, and I am not aware of plans to rename other resources.  What name do you propose? Just use the ingresscontroller's name, or prefix "ingress-" or "ingresscontroller-" or similar to the name?

Reply:
I like "ingress-", but if you want to just stay consistent with router I'm fine with that too at this point